bolt/test/AArch64/incorrect-negate-ra-state.s

```diff
@@ -0,0 +1,41 @@
+# RUN: llvm-mc -filetype=obj -triple aarch64-unknown-unknown %s -o %t.o
+# RUN: %clang %cflags  %t.o -o %t.exe -Wl,-q
+# RUN: llvm-bolt %t.exe -o %t.exe.bolt | FileCheck %s
+# check that the output is listing foo as incorrect
+# CHECK: BOLT-INFO: inconsistent RAStates in function foo
+
+# check that foo got Ignored, so it's not in the new .text section
+# llvm-objdump %t.exe -d -j .text > %t.exe.dump
+# RUN: not grep "<foo>:" %t.exe.dump
```

Comment:
I believe llvm-objdump was meant to be a RUN line and applied to `%t.exec.bolt` instead.

Currently it's not failing as`%t.exe.dump` is never created.